rb/spec/integration/selenium/webdriver/key_input_spec.rb

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
+#
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require_relative 'spec_helper'
+
+module Selenium
+  module WebDriver
+    compliant_on driver: :firefox do
+      describe W3CActionBuilder do
+        describe 'KeyInput interactions' do
+          it 'sends keys to the active element' do
+            driver.navigate.to url_for('bodyTypingTest.html')
+            keylogger = driver.find_element(id: 'body_result')
+
+            driver.action.send_keys('ab').perform
+
+            wait.until { keylogger.text.split.size == 2 }
```

Comment:
`keylogger.text.split` returns an Array, that being said your argument is still valid. `length` vs `size` is very marginal in speed especially at small sizes so it's really just user preference. I can change it to length if desired for code style

    Benchmark.measure { 10000000.times {[1,2,3,4].length}}
    => #<Benchmark::Tms:0x007f95450df020 @label="", @real=2.8369630000088364, @cstime=0.0, @cutime=0.0, @stime=0.020000000000000004, @utime=2.81, @total=2.83>

    Benchmark.measure { 10000000.times {[1,2,3,4].size}}
    => #<Benchmark::Tms:0x007f9543113b38 @label="", @real=2.8466349999653175, @cstime=0.0, @cutime=0.0, @stime=0.010000000000000009, @utime=2.8200000000000003, @total=2.83>